storage_image_product/models/product_product.py

```diff
@@ -11,6 +11,7 @@ class ProductProduct(models.Model):
     # small and medium image are here to replace
     # native image field on form and kanban
     variant_image_small_url = fields.Char(
+        # TODO: why store?
```

Comment:
store=True is probably required to avoid perf issues :thinking:

Reply:
@lmignon yeah but at this stage the thumbs are already there and the computation should not take that much.
Maybe, it has a big impact on the kanban view?
BUT storing the URL is really a pain when you restore databases across different envs.

Ahhh, while thinking again on this (right now) I think we should store the relation to the thumbnails instead of the URL.
This way the computation should should be much faster. What do you think?